java/1.2/modules/ROOT/pages/configure-java-module-acb.adoc

```diff
@@ -0,0 +1,306 @@
+= Configure the Java Module Using Anypoint Code Builder
+
+Use the Java module in your Anypoint Code Builder project to define custom Java code. To use Anypoint Code Builder to configure the Java module, create a basic integration project in Anypoint Code Builder, add connectors and other components to your Mule application to process your business logic, and then configure the attributes for the Java Module. 
+
+== Before You Begin
+
+Before creating an Anypoint Code Builder integration project, you must:
+
+* xref:anypoint-code-builder::start-acb.adoc[Set up and access the Anypoint Code Builder web or desktop IDE]
+* Have credentials to access Anypoint Platform
+* Ensure that the Visual Studio Code Extension Pack for Java is installed.
+
+[[creating-new-integration-project]]
+== Creating a New Integration Project
+
+include::connectors::partial$configure-acb-connectors-partial.adoc[tag="acb-create-integration-project"]
+
+When you create a new Anypoint Code Builder project, `src/main/java` and `/src/test/java` folders are created. For existing projects, you can manually create this folder. 
+
+
+
+[[add-components]]
+== Add Components to Your Project
+
+Add components to your integration project to build a flow:
+
+. In the *Explorer* view, open the XML file for your project if it's not already open.
+. Select *Build a Flow* from the start card in the canvas to create an empty flow:
++
+image:connectors::acb-build-flow.png["Canvas UI with starting card with the option to start from scratch, build a flow, or build a subflow"]
++
+. Change the name of the flow: 
+.. Click *Flow*. 
+.. Click the edit icon. 
++
+image:connectors::acb-change-flow-name.png["Arrow pointing to the flow name edit pencil icon"]
++
+.. Enter the flow name, and click the checkmark. 
+
+[[configure-trigger]]
+== Configure a Trigger for the Flow
+A trigger (source) starts the flow when specified conditions are met. If a connector has its own associated sources, you can use a connector-specific trigger to initiate the flow. Each connector trigger has a unique configuration.
+
+You can configure the following sources for this connector:
+
+include::connectors::partial$t-studio-partial.adoc[tags=listener-scheduler-descriptions]
```

Comment:
I see that this reference mentions these sources: 

- HTTP > Listener
- Scheduler

Are we going to add EDA too?